apps/modernization-ui/src/design-system/extendedTooltip/ExtendedTooltip.tsx

```diff
@@ -0,0 +1,32 @@
+import { Tooltip } from '@trussworks/react-uswds';
+import { ReactNode } from 'react';
+import styles from './extended-tooltip.module.scss';
+
+type Props = {
+    labelTitle: string;
+    labelText: string;
+    position?: 'bottom' | 'top' | 'left' | 'right' | undefined;
+    children: ReactNode;
+};
+
+const ExtendedTooltip = ({ labelTitle, labelText, position, children }: Props) => {
```

Comment:
suggestion:  Could you rename this to `RichTooltip`?  That is what the design team is referring to this component as.  

question: Isn't`labelText` the same as `children` here?

Reply:
1) Sure thing! 
2) Label title and label text are used in the label that gets shown/hidden in the USWDS Tooltip component, while the children like the info outlined icon here are always shown

Reply:
Ah, that makes sense